src/MongoDB.Driver.Core/Core/Authentication/SaslAuthenticator.cs

```diff
@@ -130,20 +139,21 @@ public async Task AuthenticateAsync(IConnection connection, ConnectionDescriptio
         }
 
         /// <inheritdoc/>
-        public BsonDocument CustomizeInitialIsMasterCommand(BsonDocument isMasterCommand)
+        public virtual BsonDocument CustomizeInitialIsMasterCommand(BsonDocument isMasterCommand)
         {
-            switch (_mechanism.Name)
+            return isMasterCommand;
+        }
+
+        private protected virtual BsonDocument CreateStartCommand(ISaslStep currentStep)
```

Comment:
I moved this method because a "private protected" methods feels like it shouldn't be with the other private methods. In my head, the  hierarchy for this file should be: 

```
public
protected 
private protected
private
```

Reply:
(Additionally, I had changed the accessibility of this method in order to encapsulate SCRAM related logic into the `ScramShaAuthenticator` class, per @jyemin 's request here: https://github.com/vincentkam/mongo-csharp-driver/pull/104#discussion_r407551526

Reply:
Moving code around makes diffs harder. And diffs are a powerful tool to review code changes.

Reply:
Gotcha. In the future, I can keep the method in the same position until the final round to make reviewing easier. Or leave it in the same location as long as we're not changing it from public to private or vice versa I suppose :)